src/engraving/rw/read410/tread.cpp

```diff
@@ -774,7 +774,7 @@ void TRead::read(Dynamic* d, XmlReader& e, ReadContext& ctx)
         } else if (tag == "velocity") {
             d->setVelocity(e.readInt());
         } else if (tag == "dynType") {
-            d->setDynRange(TConv::fromXml(e.readAsciiText(), DynamicRange::STAFF));
+            e.readAsciiText(); // obsolete
```

Comment:
For scores that were created in 3.6.2, then imported in 4.x, the dynamics of such scores may still have this property in them, so may still have it written in the file. So, for `<440` scores, it still makes sense to read this property, doesn't it?

Also, instead of calling `readAsciiText` and discarding the result, it might be more efficient to call `skipCurrentElement`.